libpimeval/src/pimParamsPerf.h

```diff
@@ -23,25 +23,39 @@ class pimParamsPerf
   pimParamsPerf(pimParamsDram* paramsDram);
   ~pimParamsPerf() {}
 
-  double getMsRuntimeForBytesTransfer(uint64_t numBytes) const;
-  double getMsRuntimeForFunc1(PimCmdEnum cmdType, const pimObjInfo& obj) const;
-  double getMsRuntimeForFunc2(PimCmdEnum cmdType, const pimObjInfo& obj) const;
-  double getMsRuntimeForRedSum(PimCmdEnum cmdType, const pimObjInfo& obj, unsigned numPass) const;
-  double getMsRuntimeForBroadcast(PimCmdEnum cmdType, const pimObjInfo& obj) const;
-  double getMsRuntimeForRotate(PimCmdEnum cmdType, const pimObjInfo& obj) const;
+  class perfEnergy
```

Comment:
Better to define two constructors and simplify other code.
perfEnergy() : m_msRuntime(0.0), m_mjEnergy(0.0) {}
perfEnergy(double msRuntime, double mjEnergy) : m_msRuntime(msRuntime), m_mjEnergy(mjEnergy) {}